recipe/meta.yaml

```diff
@@ -1,13 +1,12 @@
-{% set name = "PETPVC" %}
-{% set version = "1.2.11" %}
+{% set version = "1.2.12" %}
 
 package:
-  name: {{ name|lower }}
```

Comment:
This is a pure stylistic change which has nothing to do with the update, am I correct?

Reply:
If that's the case, then it should be a separate commit, and the commit message should point towards the change of policy explaining why the current form is bad.

Reply:
yes purely stylistic. 

I can revert as you wish, but one of my gripes with the example.yaml in staged recipes is that it is "optimized for only editing 1 place" but it adds alot of indirection using `{{ name }}`. notice that it is only used in 1 location.

Reply:
There is no strict policy, I've forced push my changes to cleanup the git history as requested.